extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -278,15 +295,15 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 					$res = ( is_null( $response ) ) ? false : $response;
 					break;
 				case "status":
-					$res = ( stripos( $response, self::RES_STATUS  ) !== false ) ? true : false;
+					$res = ( stripos( $response, self::RES_STATUS ) !== false ) ? true : false;
 					break;
 				case "match" :
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 17 to at most 10, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)